### PR TITLE
(Publish 02/28/2023: Morning) "What's new" distributed traces - errors and logs post

### PR DESCRIPTION
**Pull request by daynaslord**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Comment by daynaslord** [comment]
Hoping this can go out sometime tomorrow. If not, Wednesday works.

**Comment by rhetoric101** [comment]
@daynaslord thanks for submitting this "What's new" post! I just had a few questions:

* When should we publish this (Data and time window)?
* Could you add some captions under each image to explain what they mean? To do this, you'd insert <figcaption>INSERT_SOME_DESCRIPTION_HERE</figcaption>

File: src/content/whats-new/2023/02/What's New - Traces in Context Logs and Errors [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: 'Access distributed traces in context of errors inbox, APM errors, and logs'
+summary: 'The first to support metrics, events, logs, and traces'
+releaseDate: '2023-02-28'
+learnMoreLink: 'https://newrelic.com/blog/nerdlog/distributed-traces-errors-logs' 
+getStartedLink: 'https://onenr.io/07j90EyeaRO'
+---
+Distributed tracing already provides a powerful view of requests across your entire system. Now with traces in context, you will automatically get relevant information for related traces in errors inbox, APM errors, and logs. With this new capability, you can deep dive into a trace in one click without losing context to understand system complexity and performance with less toil.
+
+![Errors inbox - traces in context.png]("Error inbox - traces in context")
```
Comment (daynaslord) [inline src/content/whats-new/2023/02/What's New - Traces in Context Logs and Errors:10]: The caption: View distributed traces in the occurrences tab of errors inbox

File: src/content/whats-new/2023/02/What's New - Traces in Context Logs and Errors [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+![APM errors - traces in context.png]("APM errors - traces in context")
```
Comment (daynaslord) [inline src/content/whats-new/2023/02/What's New - Traces in Context Logs and Errors:11]: The caption: View distributed traces in the error group in the APM errors page

File: src/content/whats-new/2023/02/What's New - Traces in Context Logs and Errors [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+![Logs - traces in context.png]("Logs - traces in context")
```
Comment (daynaslord) [inline src/content/whats-new/2023/02/What's New - Traces in Context Logs and Errors:12]: The caption: View distributed traces in the log details panel

**Comment by daynaslord** [comment]
[What's New - Traces in Context Logs and Errors.zip](https://github.com/newrelic/docs-website/files/10844105/What.s.New.-.Traces.in.Context.Logs.and.Errors.zip)

@rhetoric101 - Here is the file with captions for the photos. Let me know if you have any questions or concerns.

**Comment by daynaslord** [comment]
Actually here is an updated file
[What's New - Traces in Context Logs and Errors.zip](https://github.com/newrelic/docs-website/files/10844226/What.s.New.-.Traces.in.Context.Logs.and.Errors.zip)

**Comment by rhetoric101** [comment]
I am temporarily closing this so I can kick off a new build when I reopen it.

**Review by kupsand — COMMENTED** [review]
Looks great.

Kevin Downs, New Relic PMM

**Pull request merged**